src/components/Home/TransactionCard/TransactionDetailModal/LNDetails/LNDetails.tsx

```diff
@@ -30,44 +29,40 @@ export const LNDetails: FC<LNDetailProps> = (props) => {
 
   return (
     <section className="flex flex-col py-3 my-4">
-      <article className={containerClasses}>
-        <h6 className={keyClasses}>{t("tx.txid")}</h6>
-        <p className={valueClasses}>{details.id}</p>
+      <article className="detail-container">
```

Comment:
Wondering if this could be a component (with its CSS) and coudl be used in both places

Reply:
Great idea!